lib/utils/otplease.js

```diff
@@ -1,27 +1,22 @@
 'use strict'
 
-const BB = require('bluebird')
-
-const optCheck = require('figgy-pudding')({
-  prompt: {default: 'This operation requires a one-time password.\nEnter OTP:'},
-  otp: {}
-})
+const prompt = 'This operation requires a one-time password.\nEnter OTP:'
 const readUserInfo = require('./read-user-info.js')
 
+const isOtpError = err =>
+  err.code === 'EOTP' || err.code === 'E401' && /one-time pass/.test(err.body)
```

Comment:
Silly question, does this need brackets to evaluate correctly?
```suggestion
  err.code === 'EOTP' || (err.code === 'E401' && /one-time pass/.test(err.body))
```

Reply:
It does not.  `&&` has higher priority than `||`.  In most expressions of boolean calculus, you can think of `and` (or any intersection operator) like multiplication or division, and `or` (or any union operator) like addition or subtraction. So `a || b && c || d` is like `a + b * c + d`, so it's like `a + (b * c) + d` and thus `a || (b && c) || d`.